ci/matrix.yaml

```diff
@@ -19,49 +19,52 @@ workflows:
     - {jobs: ['build'], std: 'max', cxx: ['msvc2019']}
     - {jobs: ['build'], std: 'all', cxx: ['gcc', 'clang', 'msvc']}
     # Current CTK testing:
-    - {jobs: ['test'],  project: ['libcudacxx', 'thrust'], std: 'max', cxx: ['gcc', 'clang']}
+    - {jobs: ['test'],  project: ['thrust'],     std: 'max', cxx: ['gcc', 'clang'], gpu: 't4',      sm: 'gpu'}
+    # Switching to t4 temporarily while investigating a bug in heterogeneous/barrier_abi_v2.pass.cpp
```

Comment:
Seems like this error occurred on T4, too. (Just a note that may help get this PR in to unblock CCCL development.)

https://github.com/NVIDIA/cccl/actions/runs/13022094204/job/36324973191?pr=3572#step:5:1018
```
  1: UNRESOLVED: libcu++ :: heterogeneous/barrier_abi_v2.pass.cpp (364 of 2341)
  1: ******************** TEST 'libcu++ :: heterogeneous/barrier_abi_v2.pass.cpp' FAILED ********************
```

Reply:
We're also seeing the non-abi-v2 barrier test timeout on the clang run:

```
 UNRESOLVED: libcu++ :: heterogeneous/barrier.pass.cpp
```

Reply:
I'm switching the libcudacxx testing back to v100 for now to unblock this.

Reply:
Filed https://github.com/NVIDIA/cccl/issues/3579 to track the libcudacxx issue.

Reply:
The thrust error went away on rerun, reported the issue to the runner team.

The libcudacxx builds are back on v100 for now, pending #3579.

That should be everything, assuming no more issues pop up in those last two libcu++ jobs that are running.